ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -243,120 +243,102 @@
                   disabled={{form.disabled}}
+                  aria-labelledby='toClaimHeader'
                   {{on 'change' (set-from-event claimMap 'to')}}
                   as |F|
                 >
-                  <F.Label class='sr-only'>{{t 'form.to_claim.label'}}</F.Label>
                   <F.Options>
                     {{#each this.toClaims as |toClaimOption|}}
                       <option
                         value={{toClaimOption}}
                         selected={{eq toClaimOption claimMap.to}}
-                      >{{toClaimOption}}</option>
+                      >
+                        {{toClaimOption}}
+                      </option>
                     {{/each}}
                   </F.Options>
                 </Hds::Form::Select::Field>
-
-              </row.cell>
-              <row.cell @shrink={{true}}>
-                <Rose::Button
-                  @style='warning'
-                  @iconOnly='flight-icons/svg/trash-16'
-                  title={{t 'actions.remove'}}
-                  @disabled={{if
-                    @model.isNew
-                    false
-                    (if form.isEditable false true)
-                  }}
+              </B.Td>
+              <B.Td>
+                <Hds::Button
+                  @text={{t 'actions.remove'}}
+                  @color='critical'
+                  @icon='trash'
+                  @isIconOnly={{true}}
+                  disabled={{form.disabled}}
                   {{on 'click' (fn @removeItemByIndex 'account_claim_maps' i)}}
-                >
-                  {{t 'actions.remove'}}
-                </Rose::Button>
-              </row.cell>
-            </body.row>
+                  data-test-remove-claim-map={{claimMap.from}}
+                />
+              </B.Td>
+            </B.Tr>
           {{/each}}
-
-          <body.row as |row|>
-            <row.cell>
-              <form.input
-                @type='text'
-                @name='from_claim'
+          <B.Tr>
+            <B.Td>
+              <Hds::Form::TextInput::Field
                 @value={{this.newFromClaim}}
-                @label={{t 'resources.auth-method.titles.new-from-claim'}}
+                @type='text'
+                disabled={{form.disabled}}
+                aria-labelledby='fromClaimHeader'
+                {{on 'input' (set-from-event this 'newFromClaim')}}
+                data-test-from-claim-input
               />
-            </row.cell>
-            <row.cell>
-
+            </B.Td>
+            <B.Td>
               <Hds::Form::Select::Field
                 @value={{this.newToClaim}}
+                @width='100%'
                 disabled={{form.disabled}}
+                aria-labelledby='toClaimHeader'
                 {{on 'change' (set-from-event this 'newToClaim')}}
+                data-test-to-claim-select
                 as |F|
               >
-                <F.Label class='sr-only'>{{t
```

Comment:
If we are deleting the usage of the class `sr-only`, should we delete the class declaration from the css file?

Reply:
@calcaide cleaning up in next commit

Reply:
@dvonanderson 
Can you provide an explanation of what is the outcome of this comment? You mention that the class will be removed and mark the conversation as resolved, but I do not see any change on CSS file or an answer on your end.